tests/bluetooth/tester/src/mesh.c

```diff
@@ -479,6 +485,51 @@ static void lpn_poll(u8_t *data, u16_t len)
 		   err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS);
 }
 
+static void net_send(u8_t *data, u16_t len)
+{
+	struct mesh_net_send_cmd *cmd = (void *) data;
+	struct net_buf_simple *msg = NET_BUF_SIMPLE(UINT8_MAX);
+	struct bt_mesh_msg_ctx ctx = {
+		.net_idx = net.net_idx,
+		.app_idx = BT_MESH_KEY_DEV,
+	};
+	struct bt_mesh_net_tx tx = {
+		.ctx = &ctx,
+		.xmit = bt_mesh_net_transmit_get(),
+		.sub = bt_mesh_subnet_get(net.net_idx),
+	};
+	u8_t status = BTP_STATUS_SUCCESS;
+	int err;
+
+	ctx.send_ttl = cmd->ttl;
```

Comment:
It's a bit disconcerting that the handler is given a 'len' parameter, but you never use it and instead fully trust that you can safely go and dereference cmd here like this. Is the minimum length and that "len == sizeof(*cmd) + cmd->payload_len" checked somewhere else?

Reply:
no, it is never checked...